src/theme.ts

```diff
@@ -102,8 +115,29 @@ export const darkTheme = createTheme({
     MuiPaper: {
       styleOverrides: {
         root: {
-          border: "1px solid #FFF",
           backgroundImage: "none",
+          borderColor: saddleColors.grey50,
+          borderWidth: 1,
+          borderStyle: "solid",
+        },
+      },
+    },
+    MuiDialog: {
+      styleOverrides: {
+        root: {
+          borderRadius: 10,
+        },
+        paper: {
+          borderRadius: 10,
```

Comment:
Prefer to have light and dark themes inherit all non-color overrides (like borders, spacings, fonts, etc) from a common theme, rather than duplicating code like this.

Reply:
Good idea!